crawl-ref/source/religion.cc

```diff
@@ -3905,11 +3905,26 @@ bool god_likes_spell(spell_type spell, god_type god)
     }
 }
 
+/**
+ * Does your god hate spellcasting?
+ *
+ * @param god           The god to check against
+ * @return              Whether the god hates spellcasting
+ */
 bool god_hates_spellcasting(god_type god)
 {
     return god == GOD_TROG;
 }
 
+/**
+ * Will your god put you under penance if you actually cast spell?
+ *
+ * @param spell         The spell to check against
+ * @param god           The god to check against
+ * @param fake_spell    Whether the spell is evoked, or from an innate or
```

Comment:
"Whether X is Y, or Z" sounds like you are saying Y and Z are the two options. But what you mean here is "Is it either of Y or Z?" with the implication being that it could instead be Q (i.e., an actual spell you are actually casting with the `z` command).

Clearer would be to be explicit about the two options, and say which of them is `true` and which is `false`. For example,

> `true` if the spell is evoked or from an innate or divine ability; `false` if it is a spell being cast normally.